src/frontend/apps/desk/src/features/teams/api/useTeams.tsx

```diff
@@ -2,18 +2,44 @@ import { UseQueryOptions, useQuery } from '@tanstack/react-query';
 
 import { APIList, fetchAPI } from '@/api';
 
-interface TeamResponse {
+interface Access {
+  id: string;
+  role: string;
+  user: string;
+  abilities: {
+    get: boolean;
+    patch: boolean;
+    put: boolean;
+    delete: boolean;
+    manage_accesses: boolean;
+    set_role_to: [];
+  };
+}
+
+export interface TeamResponse {
   id: string;
   name: string;
+  accesses: Access[];
 }
 
+export enum TeamsOrdering {
+  BY_CREATED_ON = 'created_at',
+  BY_CREATED_ON_DESC = '-created_at',
+}
+
+export type TeamsParams = {
+  page: number;
+  ordering?: TeamsOrdering;
+};
+
 type TeamsResponse = APIList<TeamResponse>;
 export interface TeamsResponseError {
   detail: string;
 }
 
-export const getTeams = async () => {
-  const response = await fetchAPI(`teams/`);
+export const getTeams = async ({ ordering, page }: TeamsParams) => {
+  const orderingQuery = ordering ? `&ordering=${ordering}` : '';
+  const response = await fetchAPI(`teams/?page=${page}${orderingQuery}`);
```

Comment:
is there any "nicer" way to join/stringify url params with fetch? This workaround looks error prone

Reply:
Let's ignore my comment, we would just refactor this when switching to axios